src/com/mambu/apisdk/util/RequestExecutorImpl.java

```diff
@@ -49,10 +48,19 @@
 @Singleton
 public class RequestExecutorImpl implements RequestExecutor {
 
+
+	private static final String CONTENT_TYPE_HEADER_NAME = "Content-Type";
+	private static final String AUTHORIZATION_HEADER_NAME = "Authorization";
+	private static final String USER_AGENT_HEADER_NAME = "User-Agent";
+	private final static String UTF8_charset = StandardCharsets.UTF_8.name();
```

Comment:
Can use uppercase letters for the next three names, too.

Reply:
done, thanks!